uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/util/CellContextUtilities.java

```diff
@@ -30,22 +30,66 @@
 
 public class CellContextUtilities {
 
-    public static GridBodyCellEditContext makeRenderContext(final GridWidget gridWidget,
-                                                            final BaseGridRendererHelper.RenderingInformation ri,
-                                                            final BaseGridRendererHelper.ColumnInformation ci,
-                                                            final int uiHeaderRowIndex) {
-        return makeRenderContext(gridWidget,
-                                 ri,
-                                 ci,
-                                 null,
-                                 uiHeaderRowIndex);
+    public static GridBodyCellEditContext makeCellRenderContext(final GridWidget gridWidget,
+                                                                final BaseGridRendererHelper.RenderingInformation ri,
+                                                                final BaseGridRendererHelper.ColumnInformation ci,
+                                                                final int uiRowIndex) {
+        final GridColumn<?> column = ci.getColumn();
+        final GridRenderer renderer = gridWidget.getRenderer();
+
+        final Group header = gridWidget.getHeader();
```

Comment:
I understand you made two separated specific methods, that is good.
But as it is it led to code duplication in the same class. In my Ide, lines 37-43 are the same as 93-99, and 55-59 are equals to 104-108.
Please isolate them in single methods, to ease future maintainability